### PR TITLE
Merge PR #566 from @kimw

### PR DESCRIPTION
**Pull request by wongsyrone**
- minor changes against @kimw's code
- move connection logging to newly added 'verbose logging' feature

### CODE REVIEW

**Comment by chenshaoju** [comment]
:+1: looks good to me, but someone else must approve.

File: shadowsocks-csharp/Controller/Service/TCPRelay.cs [hunk 1]
```diff
@@ -586,39 +490,58 @@ private void PipeRemoteReceiveCallback(IAsyncResult ar)
...
                 {
+                    int atyp = _connetionRecvBuffer[0];
+                    string dst_addr;
+                    int dst_port;
+                    switch (atyp)
+                    {
+                        case 1:  // IPv4 address, 4 bytes
+                            dst_addr = new IPAddress(_connetionRecvBuffer.Skip(1).Take(4).ToArray()).ToString();
+                            dst_port = (_connetionRecvBuffer[5] << 8) + _connetionRecvBuffer[6];
+                            if ( controller.GetCurrentConfiguration().isVerboseLogging ) {
```
Comment (wongsyrone) [inline shadowsocks-csharp/Controller/Service/TCPRelay.cs]: will refine "controller.GetCurrentConfiguration()" after PR #604 get merged, then we can switch to something like "config.isVerboseLogging"

**Comment by kimw** [comment]
@wongsyrone Dose it mean that I should close the #566?

**Comment by wongsyrone** [comment]
Don't worry. I will do it for you.

**Pull request merged**